scripts/run-tests.sh

```diff
@@ -7,7 +7,7 @@ scriptdir=$(realpath $(dirname "$0"))
 
 cd /root/contrail
 
-mkdir -p logs
+mkdir -p ut_logs
```

Comment:
ut_logs -> logs

Reply:
No, the idea is to have one folder logs for any activities from dev-env.  
So, the example of the structure is:
/root/contrail/logs/ut/
/root/contrail/logs/build

the taget ./run.sh logs . will just tar the whole folder /root/contrail/logs , that is bind from host into predictable place and put the tar into workspace with predictable name to be possible to collect uniformly outsite from devenv.